CRM/Event/Form/ManageEvent/EventInfo.php

```diff
@@ -187,6 +187,8 @@ public function buildQuickForm() {
     // add event title, make required if this is not a template
     $this->add('text', 'title', ts('Event Title'), $attributes['event_title'], !$this->_isTemplate);
 
+    // We must specify the entity api name when using an addSelect
+    $this->entityName = 'Event';
```

Comment:
This could be declared in the parent class. Probably simplest to put it in the class definition and not a function.